src/metabase-report-notifier/pom.xml

```diff
@@ -19,6 +19,29 @@
             <groupId>com.amazonaws</groupId>
             <artifactId>aws-lambda-java-events</artifactId>
         </dependency>
+        <dependency>
+            <groupId>software.amazon.awssdk</groupId>
+            <artifactId>s3</artifactId>
+        </dependency>
+
+        <!-- CHAMP Libraries -->
+        <dependency>
+            <groupId>com.champtitles</groupId>
+            <artifactId>metabase-report-executor</artifactId>
```

Comment:
Why is this dependency needed/good?

Reply:
@dcrough-ownum the notifier uses the KmsDecrypt library from the executor module. For later refactoring efforts we should create a separate `library` module to hold the shared code. But for the MVP I didnt feel it was necessary.

Reply:
Is the library a single class? I would rather copy the code than unnecessarily entangle these two at their beginning.